.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: "publish"
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: "Set up Node"
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          registry-url: "https://registry.npmjs.org"
+
+      - name: "Install dependencies"
+        run: npm ci
+
+      - name: "Publish package on NPM"
+        run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
issue: as of now, there is no `NPM_TOKEN` secret, so don't forget to actually add it

Reply:
yes sorry x) it's done :white_check_mark: